mantidproject/mantid
# Loader for SANS-1 at MLZ

**Description of work.**

This PR contains the loader for SANS-1 datafiles.
The loader makes it possible for Mantid to manage the files produced by the SANS-1 instrument at MLZ (ASCII files with extensions `.001`).

Pull request consists of 3 major files:
1. LoadSANS1MLZ.py (algorithm)
2. SANS1DataMLZ.py (helper file to process and structure data) 
3. LoadSANS1MLZTest.py (tests for processing and loading data into Mantid)

as well as algorithm documentation and supplemental files.

**To test:**
* Load a test file ( such as `UnitTest/D0122881.001` )
* Open the instrument viewer and check if the view works as expected (especially in 3D)

Fixes #34267.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
